logging/liblogger.go

```diff
@@ -0,0 +1,32 @@
+package logging
+
+// LibLogger is a huge `sorry` interface
+// for logging in libraries or service like packages.
+type LibLogger interface {
+	Debugf(format string, args ...interface{})
+	Infof(format string, args ...interface{})
+	Printf(format string, args ...interface{})
+	Warnf(format string, args ...interface{})
+	Warningf(format string, args ...interface{})
+	Errorf(format string, args ...interface{})
+	Fatalf(format string, args ...interface{})
+	Panicf(format string, args ...interface{})
+
+	Debug(args ...interface{})
+	Info(args ...interface{})
+	Print(args ...interface{})
+	Warn(args ...interface{})
+	Warning(args ...interface{})
+	Error(args ...interface{})
+	Fatal(args ...interface{})
+	Panic(args ...interface{})
+
+	Debugln(args ...interface{})
```

Comment:
Est-ce que sur un log on a besoin d'avoir un `ln` ? Le retour à la ligne devrait être géré déjà "out of the box" lors du logging d'un message, non ?

Reply:
effectivement